install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -160,3 +160,16 @@ spec:
               Moreover, OOM kill is expected which negatively influences the pod scheduling.
               If this happens on container level, the descheduler will not be able to detect it, as it works on the pod level.
               To fix this, increase memory of the affected node of control plane nodes.
+    - name: mcd-missing-mc
+      rules:
+        - alert: MissingMachineConfig
+          expr: |
+            mcd_missing_mc > 0
+          labels:
+            namespace: openshift-machine-config-operator
+            severity: warning
+          annotations:
+            summary: "This keeps track of Machine Config failures. Specifically a common failure on install when a rendered Machine Config is missing. Triggered when this error happens once."
+            description: >-
+              Could not find config {{ $labels.mc }} in-cluster, this likely indicates the MachineConfigs in-cluster has changed during the install process. 
+              If you are seeing this when installing the cluster, please compare the in-cluster rendered machineconfigs to s/etc/mcs-machine-config-content.json
```

Comment:
Curious, for the extra s in `s/etc/mcs-machine-config-content.json`, is that for formatting?

Reply:
fixed